checker/src/visitors.rs

```diff
@@ -535,6 +536,28 @@ impl<'a, 'b: 'a, 'tcx: 'b, E> MirVisitor<'a, 'b, 'tcx, E> {
         state_with_parameters
     }
 
+    /// Computes a fixed point over the blocks of the MIR for a promoted constant block
+    fn visit_promoted_constants_block(&mut self, function_name: &str) {
+        let mut block_indices = self.get_sorted_block_indices();
+
+        let (mut in_state, mut out_state) =
+            <MirVisitor<'a, 'b, 'tcx, E>>::initialize_state_maps(&block_indices);
+
+        // The entry block has no predecessors and its initial state is the function parameters
+        // which we omit here so that we can lazily provision them with additional context.
```

Comment:
A bit of a nit, but since in general you make your explanations fairly descriptive: I found this a bit hard to parse, maybe just some parenthesis (assuming I even did manage to parse it correctly):

```
// The entry block has no predecessors and its initial state is the function parameters
// (which we omit here so that we can lazily provision them with additional context).
```